src/configuration_env.js

```diff
@@ -38,7 +38,7 @@ export default class ConfigurationEnv {
   }
 
   static _getConfigValue(obj, key, defaultValue) {
-    return (obj && obj[key]) || defaultValue;
+    return obj && key in obj ? obj[key] : defaultValue;
```

Comment:
could you please add a unit test that would fail without this change?

Reply:
Done. And actually a very good hint as it uncovered another issue, which I fixed right away.

I now added a test with a `const` sampler and three essential assertions:
* the first one makes sure that the test still fails if no `param` is specified at all (so that my changes don't break this unintentionally)
* the second test checks that the original issue is now fixed (this one would fail prior to my commit)
* and the last one asserts that the value is parsed correctly

Would that be sufficient?